contract_isp/contract.py

```diff
@@ -215,88 +230,108 @@ def on_change_qty(self, cr, uid, ids, qty, price):
 
         return ret
 
+    def _prorata_rate(self, days_used, days_in_month):
+        """ Returns a rate to compute prorata invoices.
+        Current method is days_used / days_in_month, rounded DOWN
+        to 2 digits
+        """
+        return (100 * days_used / days_in_month) / 100.0
```

Comment:
In this case we want float division, but rounded to two decimal digits. I suppose round() would be a better call.